src/main/java/io/confluent/connect/jdbc/BulkTableQuerier.java

```diff
@@ -36,7 +36,8 @@ public BulkTableQuerier(String name) {
 
   @Override
   protected void createPreparedStatement(Connection db) throws SQLException {
-    stmt = db.prepareStatement("SELECT * FROM \"" + name + "\"");
+    String quoteString = JdbcUtils.getIdentifierQuoteString(db);
+    stmt = db.prepareStatement("SELECT * FROM " + quoteString + name + quoteString);
```

Comment:
I'm not sure this will 100% work.
Table name can be "mytable" in which case your quoting is fine, but it can also be "myschema.mytable" which is quoted "myschema"."mytable".